committee-steering/governance/sig-governance.md

```diff
@@ -7,7 +7,8 @@ In order to standardize Special Interest Group efforts, create maximum
 transparency, and route contributors to the appropriate SIG, SIGs should follow
 these guidelines:
 
-- Create a charter and have it approved according to the [SIG charter process]
+- Have an approved Charter [SIG charter process]
+- Have at least one (1) Chair and one (1) defined Tech Lead
```

Comment:
The TL requirement is a big change, while I am generally in favor it in addition to adding more roles we might want to do that change separately.